.github/workflows/build-images.yml

```diff
@@ -44,6 +45,31 @@ jobs:
           docker run $LOGIN_SERVER/$IMAGE:$SHA_TAG /bin/bash \
             -c "conda run --no-capture-output -n condaenv pytest tests/test_packages.py"
 
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+          cache: "pip"
+
+      - name: Install requirements
+        if: ${{ matrix.proc == 'cpu' }}
+        run: pip install -r requirements.txt
+
+      - name: Prep test assets
+        if: ${{ matrix.proc == 'cpu' }}
+        run: cp .github/test_assets/*.csv data/test/
+
+      - name: Test zipping submission_quickstart
+        if: ${{ matrix.proc == 'cpu' }}
+        run: |
+          make pack-quickstart
+
+      - name: Run submission in container
+        if: ${{ matrix.proc == 'cpu' }}
+        run: |
+          make CPU_OR_GPU=${{ matrix.proc }} \
+            SUBMISSION_IMAGE=$LOGIN_SERVER/$IMAGE:$SHA_TAG \
+            test-submission
```

Comment:
In the interest of good-enough, I don't think this is blocking. However, one thing to note is that the GPU job could finish running and publish the GPU image even if the CPU image fails its test. 

The way to solve this is to break the workflow up into sequential jobs, where publishing the image depends on the build and test for both images passing. There's some cleverness that's needed to cache the built image and not need to rebuild it in each stage, in order to make this happen.

Reply:
Great point. The protection against that for now is to only publish when merging into main, but this is something that we should explore further if we continue to see container issues.